tests/.eslintrc.yml

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+env:
+  jasmine: true
```

Comment:
Any specific reason why move this to a new file instead of keeping it in the `tests.js`?

Reply:
I was doing for the consistency across other repositories. For example, iOS, Android, Electron has a separate file. As well, it's a good idea to keep configurations global in a separate file, rather than inside a specific test file. Because otherwise, that would mean you would have to go to each test file and add it manually.

Reply:
Answered here apache/cordova-plugin-statusbar#143 (comment)

Reply:
Plugin tests only have one test file, all the existing eslint setups of the other core plugins have it in the `tests.js` directly, I would greatly prefer to follow the same model to keep the plugin tests as simple as possible.

Reply:
I still believe it should be in a separate file. This does not add complexity to the tests. It just keeps the configuration in a separate file. And generally, we try to do that as much as possible. Otherwise, we could also to have an `eslint` config entirely in the `package.json` file.

I would hear other's opinion on this.

Reply:
@erisu has been working on building a eslint config module so that we can have a shared ruleset  across all of our repos and when that is ready I think the eslint config file will be required to apply third-party configs.

But for the time being, I agree with janpio, we should try to keep things as consistent as possible, even though personally I would prefer eslint configs to be inside a `.eslintrc.js` file, I feel consistency is often more important when we are talking about something trivial/non-breaking.